NuGet.config

```diff
@@ -11,6 +11,7 @@
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="xamarin-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
     <add key="macios-dependencies" value="https://pkgs.dev.azure.com/xamarin/public/_packaging/macios-dependencies/nuget/v3/index.json" />
+    <add key="xamarin-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/xamarin-impl/nuget/v3/index.json" />
```

Comment:
I am confused, the same source is two lines above

Reply:
The line isn't in NuGet.config here:

https://github.com/xamarin/xamarin-macios/blob/c272040bdb3aa988c48864183500487c404713b3/NuGet.config#L6-L12

It looks like I created this branch from an earlier commit, which doesn't have the removal, so it appears duplicated.

Reply:
I clicked the "Update branch" button (to merge with the tip of main), and it's no longer duplicated.